api/v1alpha1/operatorconfig_types.go

```diff
@@ -23,9 +23,13 @@ import (
 // OperatorConfigSpec defines the desired state of OperatorConfig
 type OperatorConfigSpec struct {
 	// Operator's log level
+	//+kubebuilder:validation:Optional
+	//+kubebuilder:validation:Minimum=0
+	//+kubebuilder:validation:Maximum=2
```

Comment:
why maximum 2?

Reply:
This is operator level logging:
0 - errors only
1 - adds warning
2 - adds info

Do you have any other verbosity levels in mind for an operator?

Reply:
We can have debug as well.

Reply:
Ok. Maximum value is now 3.